src/content/docs/apis/nerdgraph/examples/nerdgraph-concurrency-tutorial.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: 'NerdGraph Tutorial: Handle Concurrency Limits'
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: Code examples for handling New Relic NerdGraph concurrency limits.
+freshnessValidatedDate: never
+---
+
+New Relic's NerdGraph API enforces a limit of **25 concurrent requests per user**. The following code examples will illustrate how to avoid hitting these limits or receiving HTTP `429` status codes.
+
+Both examples below follow the same use case, where a query is to be ran against many New Relic accounts in parallel, but respecting the NerdGraph concurrency limit.
+
+
+## Javascript
+
+One way to easily handle concurrency is to use a queuing mechanism. Below illustrates using the [async module](https://caolan.github.io/async/v3/docs.html#queue) to do so. A queue is created that has a set concurrency execution limit, requests are pushed as tasks onto that queue, and the queue is drained once all tasks are completed.
```

Comment:
I'd probably call it a "worker pool" instead of a "queuing mechanism", I think that's a more accurate description of what we're doing.